src/sagemaker/serve/model_format/mlflow/constants.py

```diff
@@ -0,0 +1,41 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+"""Holds constants used for interpreting MLflow models."""
+from __future__ import absolute_import
+
+DEFAULT_FW_USED_FOR_DEFAULT_IMAGE = "pytorch"
+DEFAULT_PYTORCH_VERSION = {
+    "py38": "1.12.1",
+    "py39": "1.13.1",
+    "py310": "2.2.0",
```

Comment:
are we providing support for python 3.11 and 3.12?

Reply:
can we add for 3.11 ?

Reply:
py311 and py312 don't have corresponding pytorch images. We can provide a default python version agnostic image in the next pr. It doesn't need to be included as part of this one